tools/driver/p4c/p4c.site.cfg

```diff
@@ -0,0 +1,34 @@
+# -*- Python -*-
+
+import os
+
+def config_preprocessor():
+    config.add_preprocessor_option("tofino", "-E -x c")
+    config.add_preprocessor_option("tofino", "-o")
+    config.add_preprocessor_option("tofino", "{}/{}.p4i".format(output_dir, source_basename))
+    config.add_preprocessor_option("tofino", source_fullname)
+
+def config_compiler():
+    config.add_compiler_option("tofino", "--nocpp")
+    config.add_compiler_option("tofino", "-o")
+    config.add_compiler_option("tofino", "{}/{}.tfa".format(output_dir, source_basename))
+    config.add_compiler_option("tofino", "{}/{}.p4i".format(output_dir, source_basename))
+
+def config_assembler():
+    config.add_assembler_option("tofino", "-vvvl")
```

Comment:
`-vvvvl` is really only for debugging the compiler/assembler combo in a way that stores the verbose output in the assembler output dir.  For futurre release, we might not want all of the `-v` options and might want to leave off the `-l` as well -- with no `-l` logging output will go to stderr, and with no `-v`, the only log output will be assembler internal `ERROR` messages which almost always mean the resulting config is corrupt.

But for now, this is a reasonble way of configuring things.